test.sh

```diff
@@ -3,10 +3,12 @@ set -e
 
 [[ "$ETH_RPC_URL" && "$(seth chain)" == "ethlive"  ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1;  }
 
-DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=1 dapp --use solc:0.6.12 build
+make build
 
 export DAPP_TEST_TIMESTAMP=$(seth block latest timestamp)
 export DAPP_TEST_NUMBER=$(seth block latest number)
 
-# LANG=C.UTF-8 hevm dapp-test --match "quick" --rpc="$ETH_RPC_URL" --json-file=out/dapp.sol.json --dapp-root=. --verbose 1
 LANG=C.UTF-8 hevm dapp-test --rpc="$ETH_RPC_URL" --json-file=out/dapp.sol.json --dapp-root=. --verbose 1
+
+# Will print the gas cost of the pass-poke hot path; useful when optimizing.
+# LANG=C.UTF-8 hevm dapp-test --rpc="$ETH_RPC_URL" --json-file=out/dapp.sol.json --dapp-root=. --verbose 2 --match test_gas_pass_poke
```

Comment:
Not sure if it's worthwhile to replicate here, but in spells mainnet the Makefile and test files are set up to accept match params.

https://github.com/makerdao/spells-mainnet/blob/master/Makefile#L6

https://github.com/makerdao/spells-mainnet/blob/master/test-dssspell.sh#L11-L15

Reply:
The different verbosity is why I didn't just make a passable arg.

Reply:
Yeah check the other one, there's an increased verbosity level on the match call.

Not a big deal, just wanted to point it out here.

Reply:
Mhm I actually like the second pattern except I want everything to work via `make`...so added a var to the Makefile for it as well (see latest changes).

Reply:
Specifically:
https://github.com/makerdao/univ2-lp-oracle/pull/41/commits/64fb52c7bc2f3400168a579271c304e6e637762b

Reply:
we've used this pattern elsewhere as `match` (case-sensitive)

Don't have a strong opinion on which way to go, but we should be consistent.

Reply:
Personally I like capitalizing Makefile/environment variables, e.g. `TEST_FLAGS` in `dss`, and I think capitalizing is the more common pattern overall (e.g. all the DappTools envvars are named in all caps). So I'd argue for capitalizing it elsewhere :upside_down_face: .